### DISCUSSION TITLE
Drag tab to create split.

### DISCUSSION THREAD

**Discussion by jacakira** [ghostty-org/ghostty#2970] [Feature Requests, Ideas] [answered]
In iTerm2, one is able to drag a tab to create a split (as shown in video below) and place it in a certain position.

https://github.com/user-attachments/assets/404ab5ac-df6f-4cea-8c60-03b1115d88f6

Is it possible for the same functionality to be replicated in Ghostty?

As a side note, I find that the red highlight (as seen in the video) for potential split areas in iTerm is very ugly, and were such a feature to be implemented in Ghostty, it would be nice if the shading were more tasteful.

**Comment by pluiedev** [comment, marked as answer]
This would probably be a part of moving splits in general (#1525)
